packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -71,7 +71,7 @@ export class TileDoctype extends Doctype {
 
         const metadata: GenesisHeader = params.metadata || { controllers: [] }
         if (!metadata.controllers || metadata.controllers.length === 0) {
-            if (params.content && context.did) {
```

Comment:
Was there some reason for this requirement I'm not seeing @oed?

Reply:
I guess no. As long as the genesis commit is not signed when content is undefined we should be all good.